protected/humhub/modules/post/controllers/PostController.php

```diff
@@ -101,4 +103,16 @@ public function actionEdit($id)
         ]);
     }
 
+    public function actionForm()
+    {
+        if (!$this->contentContainer->getPermissionManager()->can(CreatePost::class)) {
```

Comment:
Why not use new style?  `if ((new Post($this->contentContainer))->canEdit())`

Reply:
Ok, done in commit https://github.com/humhub/humhub/pull/5827/commits/33cca19cc872d6bdde229d426b2d47a60b82fb74